### PR TITLE
v1.4.0

### PR DESCRIPTION
**Pull request by bugsounet**

### CODE REVIEW

**Comment by bugsounet** [comment]
tell me if ok for you ?

File: node_helper.js [hunk 1]
```diff
@@ -9,61 +9,66 @@ const Spotify = require("./Spotify.js")
...
-      this.spotifyConfigurations = updateOldSingleSpotifyConfigurationToNewMultipleSpotifyConfiguration(parsedConfigurations);
-      this.spotifyConfigurations.forEach(configuration => {
-        this.spotifies.push(new Spotify(configuration, this.config.debug));
-      });
+  doSpotifyConfig: function (configuration, account) {
+    if (!isNaN(account) && Array.isArray(configuration)) {
+      return configuration[account] // only wanted account or first
+    }
+    if (Array.isArray(configuration)) {
+      var found
```
Comment (humbertovirtudes) [inline node_helper.js]: nit: let instead of var

File: node_helper.js [hunk 2]
```diff
@@ -9,61 +9,66 @@ const Spotify = require("./Spotify.js")
...
-      setTimeout(() => {
-        this.findCurrentSpotify();
-      }, this.config.updateInterval);
-    } else {
-      setTimeout(() => {
-        this.updatePulse()
-      }, this.config.updateInterval)
+    else return console.log("[SPOTIFY] ERROR: spotify.config.json file missing !")
+    this.updatePulse().then(() => {
+      console.log("[SPOTIFY] Started with Account:", (this.spotify.config.USERNAME ? this.spotify.config.USERNAME : "default"))
```
Comment (humbertovirtudes) [inline node_helper.js]: do we need this log?

**Review by humbertovirtudes — APPROVED** [review]
looks good to me! Just two small comments.

Also, should we add support to display the spotify account name somewhere?

**Comment by humbertovirtudes** [comment]
96798763_194217508266373_7969196132783882240_n
btw i'm pretty much done with the new "minimalist_bar" layout. It looks great as a fullwidth item either on the top or bottom of the MM.

just making some css tweaks & ensuring code is reused as much as possible 

Should have a PR by tomorrow

File: node_helper.js (hunk 1 above)
Comment (bugsounet) [reply node_helper.js]: yes of course !
let found

File: node_helper.js (hunk 2 above)
Comment (bugsounet) [reply node_helper.js]: for error yes! , i should be displayed

maybe:

if (this.config.debug) console.log("[SPOTIFY] Started with Account:", (this.spotify.config.USERNAME ? this.spotify.config.USERNAME : "default"))

**Comment by bugsounet** [comment]
so, we will wait to merge it

after, if you want you can display account name

**Pull request merged**